### ISSUE TITLE
Not supported proxy scheme socks5

### ISSUE THREAD

**Issue by alexanderkent** [ralphwetzel/theonionbox#27]
BOX | Onionoo: Failed querying 'http://onionoorcazzotwa.onion/weights' -> Not supported proxy scheme socks5

Yet,  curl -v --socks5 52.175.226.63:9050 https://check.torproject.org | grep -i "Your IP"
works!

Any ideas?  Tried python2.7 and python3.

**Comment by ralphwetzel** [comment]
Hi!
Most probably you get this error message because the `requests` package is outdated.
[This hint](https://github.com/ralphwetzel/theonionbox/blob/devel/README_v4.md#i-receive-a-socks5h-not-supported-warning-what-shall-i-do) (Q&A at the end of the file; link might not work as expected) perhaps can help you to solve the issue.
Greetings, Ralph

**Comment by ralphwetzel** [comment]
And ... there seems to be a bug in the 3.2.1 code:
onionoo.py, line 612:
```
proxies = {}
if proxy is not None:
    proxies = {
        'http': 'socks5://' + proxy,
        'https': 'socks5://' + proxy
    }
```
should be
```
proxies = {}
if proxy is not None:
    proxies = {
        'http': 'socks5h://' + proxy,
        'https': 'socks5h://' + proxy
    }
```
This will be fixed with the next release (4.0).
Ralph

**Comment by alexanderkent** [comment]
Thanks that partially helped. The error went away but the metrics still don't load for me.
